src/main/fc/parameter_names.h

```diff
@@ -211,16 +211,26 @@
 
 #ifdef USE_MAG
 #define PARAM_NAME_GPS_RESCUE_USE_MAG "gps_rescue_use_mag"
-#endif
-#endif
+#endif // USE_MAG
+#endif // USE_GPS_RESCUE
 
 #ifdef USE_GPS_LAP_TIMER
 #define PARAM_NAME_GPS_LAP_TIMER_GATE_LAT "gps_lap_timer_gate_lat"
 #define PARAM_NAME_GPS_LAP_TIMER_GATE_LON "gps_lap_timer_gate_lon"
 #define PARAM_NAME_GPS_LAP_TIMER_MIN_LAP_TIME "gps_lap_timer_min_lap_time_s"
 #define PARAM_NAME_GPS_LAP_TIMER_GATE_TOLERANCE "gps_lap_timer_gate_tolerance_m"
 #endif // USE_GPS_LAP_TIMER
-#endif
+#endif //
```

Comment:
```suggestion
#endif // USE_GPS_LAP_TIMER
```

Reply:
I seemed that we need an endif for USE_GPS_LAP_TIMER and another for USE_GPS ?

Reply:
Sorry I meant USE_GPS indeed